src/AppBundle/Resources/views/book/edit_contribution.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Book Contribution Edit{% endblock %}
+
+{% block body %}
+    <h1>Book Contribution Edit</h1>
+
+    {{ form_start(edit_form) }}
+    {{ form_widget(edit_form) }}
+    <div class='btn-group'>
+        <input type="submit" value="Update" class="btn btn-primary" />
+        <a href="{{ path('book_show_contributions', { 'id': contribution.publication.id }) }}" class="btn btn-warning">Cancel</a>
```

Comment:
You could use `contribution.getPublicationId()` here instead of `contribution.publication.id` if you wanted.